src/components/CertificateDropZone/Views/UnverifiedView.js

```diff
@@ -1,151 +1,74 @@
 import PropTypes from "prop-types";
 import Link from "next/link";
+import { get } from "lodash";
+import { TYPES, MESSAGES } from "../../../constants/VerificationErrorMessages";
 import css from "./viewerStyles.scss";
 
-const View = ({
-  resetData,
-  storeStatus,
-  hashStatus,
-  issuedStatus,
-  issuerIdentityStatus,
-  notRevokedStatus
-}) => {
-  /* Array of error messages with priority of error messages determined by a stack. 
-  Error messages are first placed into the stack and the error message with the highest priority is popped off the stack
-  and displayed.
-  
-  The priority of error messages are as follows:
-  1. Invalid store
-  2. Tampered
-  3. Not issued
-  4. Revoked */
-  const errorMessages = [
-    {
-      title: "Certificate revoked",
-      message:
-        "This certificate has been revoked by your issuing institution. Please contact your issuing institution for more details.",
-      error: !notRevokedStatus.verified
-    },
-    {
-      title: "Certificate not issued",
-      message:
-        "This certificate cannot be found. Please contact your issuing institution for help or issue the certificate before trying again.",
-      error: !issuedStatus.verified
-    },
-    {
-      title: "Certificate has been tampered with",
-      message:
-        "The contents of this certificate are inaccurate and have been tampered with.",
-      error: !hashStatus.verified
-    },
-    {
-      title: "Certificate issuer identity invalid",
-      message: "This certificate was issued by an invalid issuer.",
-      error: !issuerIdentityStatus.verified
-    },
-    {
-      title: "Certificate store address is invalid",
-      message:
-        "Please check that you have a valid smart contract with us and a correct certificate store address before proceeding.",
-      error: !storeStatus.verified
-    }
-  ];
+const DetailedErrors = ({ verificationStatus }) => {
+  const errors = [];
+  if (!get(verificationStatus, "hash.checksumMatch")) errors.push(TYPES.HASH);
+  if (!get(verificationStatus, "issued.issuedOnAll")) errors.push(TYPES.ISSUED);
+  if (get(verificationStatus, "revoked.revokedOnAny", true))
```

Comment:
why default to true ?

Reply:
All the above defaults to the error output